src/components/Sectionfour.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import eleven from '../assets/eleven.png';
+import twelve from '../assets/twelve.png';
+import thirteen from '../assets/thirteen.png';
+import ten from '../assets/ten.png';
+
+const Sectionfour = () => {
+  return (
+      <div>
+          <div className=' flex lg:flex-row gap-6 items-center justify-center px-6 cursor-pointer '>
+          
+          <div className='grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-6'>
+            
+              <div className=''>
+              <img src={eleven} alt="" />
+              <div className='text-left'>
+                    <p>IGURE</p>
+                    <p className='font-bold'>GREEN MUSCLE FIT POLO SHIRT</p>
+                    <p><s>$229.00</s> <span className='orange-text'>$129.00</span></p>
+              </div>
+                      
+          </div>
+          <div className=''>
+              <img src={twelve} alt=" " />
```

Comment:
would suggest adding an alt text to all images

Reply:
got it